lib/hanami/config/logger.rb

```diff
@@ -135,6 +141,8 @@ def initialize(env:, app_name:)
         config.formatter = case env
                            when :production
                              :json
+                           else
+                            :application
```

Comment:
It's prefixed with the application name but I agree that having a dedicated formatter feels like too much. I can just add a configured string formatter for Hanami purposes *to Hanami* instead.

> Could we just default to JSON logging everywhere?

I don't like JSON logging in development and test envs. It is easier to parse string logs with your eyes. We should absolutely default to JSON in production envs though (which we do).

Reply:
> It's prefixed with the application name but I agree that having a dedicated formatter feels like too much. I can just add a configured string formatter for Hanami purposes to Hanami instead.

Yeah, I think this would be a better arrangement 👍🏼

Reply:
@timriley done via 38e1676537c3c5d0862920902952d19ed0b2b271